packages/tasit-sdk/src/helpers/DecentralandUtils.test.js

```diff
@@ -65,34 +65,51 @@ describe("DecentralandUtils", () => {
 
     const estateContractAddress = estate.getAddress();
 
-    const sellerEstateIds = await getEstateIdsOf(sellerAddress);
+    const sellerEstates = await _getEstatesOf(sellerAddress);
     const sellerEstateBalance = await estate.balanceOf(sellerAddress);
-    const { length: sellerEstateIdsLength } = sellerEstateIds;
-    expect(`${sellerEstateIdsLength}`).to.equal(`${sellerEstateBalance}`);
+    expect(sellerEstates).to.have.lengthOf(sellerEstateBalance);
 
-    const sellerParcelIds = await getParcelIdsOf(sellerAddress);
+    const sellerParcels = await _getParcelsOf(sellerAddress);
     const sellerParcelBalance = await land.balanceOf(sellerAddress);
-    const { length: sellerParcelIdsLength } = sellerParcelIds;
-    expect(`${sellerParcelIdsLength}`).to.equal(`${sellerParcelBalance}`);
+    expect(sellerParcels).to.have.lengthOf(sellerParcelBalance);
 
-    const estateParcelIds = await getParcelIdsOf(estateContractAddress);
+    const estateParcels = await _getParcelsOf(estateContractAddress);
     const estateParcelBalance = await land.balanceOf(estateContractAddress);
-    const { length: estateParcelIdsLength } = estateParcelIds;
-    expect(`${estateParcelIdsLength}`).to.equal(`${estateParcelBalance}`);
+    expect(estateParcels).to.have.lengthOf(estateParcelBalance);
 
-    const sellerAsset = await getAssetsOf(sellerAddress);
-    const sellerAssetIds = sellerAsset.map(asset => asset.id);
+    const sellerAssets = await getAssetsOf(sellerAddress);
+    const sellerAssetIds = sellerAssets.map(a => a.id);
+    const sellerEstateIds = sellerEstates.map(e => e.id);
+    const sellerParcelIds = sellerParcels.map(p => p.id);
     expect(sellerAssetIds).to.have.members([
       ...sellerParcelIds,
       ...sellerEstateIds,
     ]);
   });
 
   it("should get empty array from an address without assets", async () => {
-    const estateIds = await getEstateIdsOf(ephemeralAddress);
-    const parcelIds = await getParcelIdsOf(ephemeralAddress);
+    const estateIds = await _getEstatesOf(ephemeralAddress);
+    const parcelIds = await _getParcelsOf(ephemeralAddress);
 
     expect(estateIds).deep.equal([]);
     expect(parcelIds).deep.equal([]);
   });
+
+  it("should expose transactionHash", async () => {
+    const sellerAssets = await getAssetsOf(sellerAddress);
+    expect(sellerAssets).to.not.be.empty;
+
+    for (let asset of sellerAssets) {
+      let { transactionHash } = asset;
+      expect(transactionHash).to.be.ok;
```

Comment:
Make a TODO to assert something more specific? Or is it non-deterministic for a given test run? Maybe if we stored the hash from the action after doing a transfer, then did `getAssetsOf` and compared that the `transactionHash` from that function with the action's hash?

Reply:
Make sense, I will do that on `Decentraland.test.js` test suite.

Reply:
Okay, sounds good. Thanks!

Reply:
Done

Reply:
Thanks! Resolving